nrf_wifi/bus_if/bal/inc/bal_ops.h

```diff
@@ -13,67 +13,167 @@
 #define __BAL_OPS_H__
 
 /**
- * struct nrf_wifi_bal_ops - Ops to be provided by a particular bus
- *                           implementation.
- * @init:
- * @deinit:
- * @dev_init:
- * @dev_deinit:
- * @read_word:
- * @write_word:
- * @read_block:
- * @write_block:
- * @dma_map:
- * @dma_unmap:
+ * @brief Ops to be provided by a particular bus implementation.
+ *
+ * This structure defines the operations that need to be implemented
+ * by a specific bus implementation.
  */
 struct nrf_wifi_bal_ops {
+	/**
+	 * @brief Initializes the bus.
```

Comment:
```suggestion
	 * @brief Initialize the bus.
```
Use always the imperative form of the verb in the function description. Change this in all instances.